code/modules/clothing/gloves/_gloves.dm

```diff
@@ -9,6 +9,7 @@
 	attack_verb_continuous = list("challenges")
 	attack_verb_simple = list("challenge")
 	var/transfer_prints = FALSE
+	var/can_be_cut = FALSE
```

Comment:
Could you move these new variables to the bottom of the list, after the redefined ones?
Also document the new variable you are adding.
Something like this:
```nix
/// Boolean. If TRUE this glove will produce a fingerless version when cut by wirecutters.
var/can_be_cut = FALSE
```